README.md

```diff
@@ -1 +1,162 @@
-# hello-microvers
\ No newline at end of file
+# hello-microvers
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
You seemed to have forgotten to remove the Microverse logo as requested by the previous reviewer. Kindly do so or replace it with your personal logo.